src/test/regress/expected/citus_local_tables.out

```diff
@@ -496,6 +513,12 @@ CREATE TABLE local_table_4 (
   a int unique references citus_local_table_1(a),
   b int references local_table_4(a));
 NOTICE:  executing the command locally: SELECT worker_apply_inter_shard_ddl_command (xxxxx, 'citus_local_tables_test_schema', xxxxx, 'citus_local_tables_test_schema', 'ALTER TABLE citus_local_tables_test_schema.local_table_4 ADD CONSTRAINT local_table_4_a_fkey FOREIGN KEY (a) REFERENCES citus_local_tables_test_schema.citus_local_table_1(a)')
+NOTICE:  creating a new table for test_index_schema.testindex
+NOTICE:  Moving the data of test_index_schema.testindex
+NOTICE:  executing the command locally: SELECT a, b FROM test_index_schema.testindex_1504035 testindex
+NOTICE:  Dropping the old test_index_schema.testindex
+NOTICE:  executing the command locally: DROP TABLE IF EXISTS test_index_schema.testindex_xxxxx CASCADE
+NOTICE:  Renaming the new table to test_index_schema.testindex
```

Comment:
Those notice messages are due to something that we recently introduced via https://github.com/citusdata/citus/pull/4538.
Your citus local table gets undistributed when an unrelated create table commands drop some constraints.

In the end can you replace:
`SELECT create_citus_local_table('test_index_schema.testindex');`
with something like
`ALTER TABLE test_index_schema.testindex ADD CONSTRAINT fkey_to_dummy_ref FOREIGN KEY (a) REFERENCES dummy_reference_table(a);`